src/LabelSelector/LabelChip/LabelChip.test.tsx

```diff
@@ -77,4 +77,46 @@ describe("LabelChip", () => {
     // there shouldn't be a cancel icon
     expect(screen.queryByTestId("CancelIcon")).not.toBeInTheDocument();
   });
+  // test that the selected icon is rendered when selected is true
+  it("renders selected icon", () => {
+    render(<LabelChip label="Label" selected />);
+
+    // get the selected icon
+    const selectedIcon = screen.getByTestId("DoneIcon");
+
+    // check that the selected icon is rendered
+    expect(selectedIcon).toBeInTheDocument();
+  });
+  // test that the selected icon is not rendered when selected is false
+  it("does not render selected icon", () => {
+    render(<LabelChip label="Label" selected={false} />);
+
+    // there shouldn't be a selected icon
+    expect(screen.queryByTestId("DoneIcon")).not.toBeInTheDocument();
+  });
+  // renders tooltip on hover of label chip
+  it("renders tooltip on hover", async () => {
```

Comment:
Your new tests are throwing a bunch of warnings;

https://github.com/IPG-Automotive-UK/react-ui/actions/runs/10370550853/job/28708693334#step:6:237